src/presentation/http/index.ts

```diff
@@ -16,6 +17,11 @@ export default class HttpServer implements Server {
     logger: appServerLogger,
   });
 
+  /**
+   * Note route
+   */
+  private router = new Router(this.server);
```

Comment:
do you need to manually pass a Fastify instance? Looks like it's already passed via `register`

Reply:
Now i am passing it to the method